doc/source/whatsnew/v2.0.1.rst

```diff
@@ -33,6 +33,7 @@ Bug fixes
 - Bug in :meth:`ArrowDtype.__from_arrow__` not respecting if dtype is explicitly given (:issue:`52533`)
 - Bug in :meth:`DataFrame.max` and related casting different :class:`Timestamp` resolutions always to nanoseconds (:issue:`52524`)
 - Bug in :meth:`Series.describe` not returning :class:`ArrowDtype` with ``pyarrow.float64`` type with numeric data (:issue:`52427`)
+- Fixed bug in :func:`first` when used with a :class:`DateOffset` (:issue:`45908`)
```

Comment:
there is no pd.first.  Maybe `:meth:DataFrame.first`?

Reply:
Change done